tools/Microsoft.AspNet.Server.Kestrel.LibuvCopier/project.json

```diff
@@ -26,6 +17,7 @@
         "System.Threading": "4.0.11-*",
         "System.IO.FileSystem": "4.0.1-*"
       }
-    }
+    },
+    "dnx451": { }
```

Comment:
Is this needed? this tool is run only with dotnet and dotnet is always CoreClr...